app/views/hyrax/base/_form_child_work_relationships.html.erb

```diff
@@ -10,14 +10,29 @@ HTML Properties:
     [data-behavior="child-relationships"] : allows the javascript to be initialized
     data-param-key : the parameter key value for this model type
 %>
-<div class="form-group multi_value optional managed">
-  <%= link_to "Attach New Work", polymorphic_path([main_app, :new, :hyrax, :parent, curation_concern.model_name.singular], parent_id: curation_concern.id), target: "_blank", class: 'btn btn-primary' %>
-
-  <table class="table table-striped related-files" data-behavior="child-relationships" data-param-key="<%= f.object.model_name.param_key %>">
+<div class="form-group" data-behavior="child-relationships" data-param-key="<%= f.object.model_name.param_key %>">
+  <div class="form-inline">
+      <%= f.label :find_child_work %>
+      <%= f.input_field :find_child_work,
+                        prompt: :translate,
+                        autocomplete: 'off',
+                        data: {
+                          autocomplete: 'work',
+                          'autocomplete-url' => Rails.application.routes.url_helpers.qa_path + '/search/find_works',
+                          'exclude-work': f.object.model.id # exclude this item from the result set.
+                        } %>
+      <a class="btn btn-primary" data-behavior="add-relationship">Add</a>
+  </div>
+  <div class="message has-warning hidden"></div>
+  <div style="margin-top: 10px">
```

Comment:
Are we ok with inline styling here, or should this be classed?

Reply:
I don't know that we want another class just to bump a button down 10px.  Do you have a preference?